zulipterminal/helper.py

```diff
@@ -711,3 +715,88 @@ def suppress_output() -> Iterator[None]:
     finally:
         os.dup2(stdout, 1)
         os.dup2(stderr, 2)
+
+
+@asynch
+def process_media(controller: Any, link: str) -> None:
+    """
+    Helper to process media links.
+    """
+    if not link:
+        controller.report_error("The media link is empty")
+        return
+
+    show_download_status = partial(
+        controller.view.set_footer_text, "Downloading your media..."
+    )
+    media_path = download_media(controller, link, show_download_status)
+    tool = ""
+
+    # TODO: Add support for other platforms as well.
+    if PLATFORM == "WSL":
+        tool = "explorer.exe"
+        # Converting absolute path to WSL path
+        media_path = f'`wslpath "{media_path}"`'
```

Comment:
What is interpreting these backticks? What else is it interpreting? Something is definitely wrong here.

Reply:
Yeah. It is a bad way of nesting bash commands. One should instead use `$()` I think. As I said in the stream, we'd potentially not need this now.

Reply:
No, my point is there should not _be_ a Bash. Backticks should not be interpreted here. If they _were_ being interpreted here—even if we’re no longer _using_ them—that indicates a deep problem that we need to investigate, because `media_path` could itself contain backticks or other characters that we really don’t want to be interpreted.

The same problem applies to `$()`, `""`, or any other kind of “quote” construct. The first rule of quoting is that you need to ask, _what if the quoting characters are present inside the string you’re quoting?_

Reply:
Makes sense. Thanks for clarifying :+1: